discovery-provider/integration_tests/queries/test_get_attestation.py

```diff
@@ -123,6 +126,149 @@ def test_get_attestation(app):
                 )
 
 
+def test_get_attestation_weekly_pool_exhausted(app):
+    with app.app_context():
+        db = get_db()
+
+    entities = {
+        "users": [
+            {"user_id": 1, "wallet": "0x38C68fF3926bf4E68289672F75ee1543117dDAAA"},
+            {"user_id": 2, "wallet": "0x38C68fF3926bf4E68289672F75ee1543117dD9B3"},
+            {"user_id": 3, "wallet": "0xDEADBEEFDEADBEEFDEADBEEFDEADBEEFDEADBEEF"},
```

Comment:
lol

Reply:
Oh wow what a throwback. Haven't deadbeef-ed anything in a bit.